wordle-web/pages/wordEditor.vue

```diff
@@ -0,0 +1,364 @@
+          </td>
+          <td class="text-center text-h8">
+            {{ word.isCommonWord ? 'Yes' : 'No' }}
+          </td>
+          <td class="text-center text-h8">
+            <v-row class="mt-0">
+              <v-col cols="3">
+                <v-tooltip
+                  text="Tooltip"
+                  :disabled="tokenService?.isLoggedIn()">
+                  <template v-slot:activator="{ props }">
+                    <div class="text-center mx-0" v-bind="props">
+                      <v-switch
+                        :color="tokenService?.isLoggedIn() ? 'primary' : ''"
+                        :disabled="!tokenService?.isLoggedIn()"
+                        v-model="word.isCommonWord"
+                        @update:modelValue="
+                          isCommonWordChanged[index] =
+                            !isCommonWordChanged[index]
+                        "></v-switch>
+                    </div>
+                  </template>
+                  <span>You must be logged in to change common word flag</span>
+                </v-tooltip>
+              </v-col>
+              <v-col class="mt-2" cols="4">
+                <v-tooltip
+                  text="Tooltip"
+                  :disabled="isOlderThanTwentyOne && isMotU">
+                  <template v-slot:activator="{ props }">
+                    <div class="text-center mx-0" v-bind="props">
+                      <v-btn
+                        :color="isOlderThanTwentyOne && isMotU ? 'error' : ''"
+                        :disabled="!(isOlderThanTwentyOne && isMotU)"
+                        :variant="
+                          isOlderThanTwentyOne && isMotU ? 'outlined' : 'plain'
+                        "
+                        icon="mdi-trash-can-outline"
+                        size="35"
+                        @click="deleteWord(word)"></v-btn>
+                    </div>
+                  </template>
+                  <span
+                    >You must be Master of the Universe and 21 or older to add
+                    words</span
+                  >
+                </v-tooltip>
+              </v-col>
+              <v-col class="mt-2" cols="4">
+                <v-btn
+                  :variant="isCommonWordChanged[index] ? 'elevated' : 'plain'"
+                  @click="
+                    isCommonWordChanged[index] = false;
+                    saveCommonWordFlag(word.word, word.isCommonWord);
+                  "
+                  :color="isCommonWordChanged[index] ? 'primary' : ''">
+                  Save
+                </v-btn>
+              </v-col>
+            </v-row>
+          </td>
+        </tr>
+      </tbody>
+    </v-table>
+    <v-pagination
+      @prev="pageDisplay(wordPerPage, wordText, pageNum - 1)"
+      @next="pageDisplay(wordPerPage, wordText, pageNum + 1)"></v-pagination>
+    <!--<v-pagination v-model="curretnPage":length="pageCount()" :totalVisible="7"></v-pagination>-->

```

Comment:
In my opinion, I would use a v-model to track the current page. and then use the number of words as length property and update it using pageDisplay function using an @v-model:update or an @input event.

Reply:
Thank you for the advise will try and Implement